src/main/java/taxi/controller/IndexController.java

```diff
@@ -10,6 +10,6 @@ public class IndexController extends HttpServlet {
     @Override
     public void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        req.getRequestDispatcher("/WEB-INF/views/index.jsp").forward(req, resp);
+        req.getRequestDispatcher("/WEB-INF/views/pages/index.jsp").forward(req, resp);
```

Comment:
I'm not sure 'pages' is an appropriate folder here. If so then why aren't your other jsps placed in this folder?